solr/core/src/java/org/apache/solr/util/ExportTool.java

```diff
@@ -490,7 +491,7 @@ class CoreHandler {
       }
 
       boolean exportDocsFromCore() throws IOException, SolrServerException {
-        HttpSolrClient client = new HttpSolrClient.Builder(baseurl).build();
+        SolrClient client = new HttpSolrClient.Builder(baseurl).build();
```

Comment:
Up to you, but if you can easily also do try-with-resources pattern as well, that'd be nice.

Reply:
@dsmiley I just updated this to what I think is try with resrouces...   Does this mean the `.close()` gets magically called?  how does it know to do that?

Reply:
never mind, read more detail in https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
wonder if we should have a seperate issue to refactor to this!  we have this try/finally structure everywhere!